libs/application/template-api-modules/src/lib/modules/templates/accident-notification/accident-notification-v2.utils.ts

```diff
@@ -0,0 +1,431 @@
+import { AccidentNotificationAttachment } from './types/attachments'
+import { getValueViaPath } from '@island.is/application/core'
+import {
+  AccidentDetailsV2,
+  AccidentNotificationAnswers,
+  AccidentTypeEnum,
+  ApplicantV2,
+  CompanyInfoV2,
+  FishermanWorkplaceAccidentLocationEnum,
+  FishingShipInfoV2,
+  GeneralWorkplaceAccidentLocationEnum,
+  HomeAccidentV2,
+  InjuredPersonInformationV2,
+  RepresentativeInfoV2,
+  StudiesAccidentTypeEnum,
+  WhoIsTheNotificationForEnum,
+  WorkAccidentTypeEnum,
+  WorkMachineV2,
+} from '@island.is/application/templates/accident-notification'
+import { YesOrNo } from '@island.is/application/types'
+import {
+  MinarsidurAPIModelsAccidentReportsAccidentReportDTO,
+  MinarsidurAPIModelsAccidentReportsReporterDTO,
+  MinarsidurAPIModelsAccidentReportsInjuredDTO,
+  MinarsidurAPIModelsAccidentReportsAccidentDTO,
+  MinarsidurAPIModelsAccidentReportsEmployerDTO,
+  MinarsidurAPIModelsAccidentReportsClubDTO,
+  MinarsidurAPIModelsAccidentReportsAccidentReportAttachmentDTO,
+  MinarsidurAPIModelsAccidentReportsReporterDTOReportingForEnum,
+} from '@island.is/clients/icelandic-health-insurance/rights-portal'
+
+export const applicationToAccidentReport = (
+  answers: AccidentNotificationAnswers,
+  attachments: Array<AccidentNotificationAttachment>,
+): MinarsidurAPIModelsAccidentReportsAccidentReportDTO => {
+  return {
+    reporter: getReporter(answers),
+    injured: getInjured(answers),
+    accident: getAccident(answers),
+    employer: getEmployer(answers),
+    club: getClub(answers),
+    attachments: getAttachments(attachments),
+  }
+}
+
+const reportingForMap = {
+  [WhoIsTheNotificationForEnum.ME]:
+    MinarsidurAPIModelsAccidentReportsReporterDTOReportingForEnum.NUMBER_1,
+  [WhoIsTheNotificationForEnum.JURIDICALPERSON]:
+    MinarsidurAPIModelsAccidentReportsReporterDTOReportingForEnum.NUMBER_2,
+  [WhoIsTheNotificationForEnum.POWEROFATTORNEY]:
+    MinarsidurAPIModelsAccidentReportsReporterDTOReportingForEnum.NUMBER_3,
+  [WhoIsTheNotificationForEnum.CHILDINCUSTODY]:
+    MinarsidurAPIModelsAccidentReportsReporterDTOReportingForEnum.NUMBER_4,
+}
+
+const whoIsTheNotificationForToDTO = (who: WhoIsTheNotificationForEnum) => {
+  return (
+    reportingForMap[who] ||
+    MinarsidurAPIModelsAccidentReportsReporterDTOReportingForEnum.NUMBER_1
+  )
+}
+
+/*
+ * type can be 4: Íþróttaslys, 6: Vinnuslys, 7: Heimilistrygging, 8: Björgunarmenn, 9: Nemendur við iðnnám
```

Comment:
Consider making this line a bit more readable, something like
```suggestion
 * type can be one of the following:
    4: Íþróttaslys, 
    6: Vinnuslys, 
    7: Heimilistrygging, 
    8: Björgunarmenn, 
    9: Nemendur við iðnnám
```